code/datums/ai_law_sets.dm

```diff
@@ -25,7 +25,7 @@
 
 /******************** Nanotrasen/Malf ********************/
 /datum/ai_laws/nanotrasen
-	name = "NT Default"
+	name = "NT Recommended"
```

Comment:
I think "StationGuard" sounds good considering law one.
"Safeguard: Protect your assigned space station to the best of your abilities. It is not something we can easily afford to replace."